draft-quic-atsss-reqs.mkd

```diff
@@ -194,6 +196,8 @@ the different access networks even if the server does not support it.
 The UE and the Proxy exchange, using the 5G control plane, rules that specify
 which flows are eligible to the ATSSS service.
 
+(SEO's comment : we should check the "ATSSS rules from network node" in Figure 1 can directly interact with and provision the rules to the Proxy denpending on the location of the Proxy. It's okay if the Proxy is inside the 3GPP CN (Core NW), but we need to implement exposure function e.g., SCEF and/or NEF to communicate the Proxy with other 3GPP nodes and UE.)
```

Comment:
I feel those considerations are too early for the first version of the document. It's unclear to me what role will those ATSSS rules play when defining the requirements.